submissions/damilolaSalami/domManipulation/signUpform.js

```diff
@@ -0,0 +1,42 @@
+let submitform = document.getElementById("validateForm");
```

Comment:
You can use `const` instead of `let` the variables are not going to reassigned. You should separate each variable with a comma instead of a semicolon @queensaldam